forge/test/operators/utils/compat.py

```diff
@@ -22,6 +22,85 @@ class TestDevice:
     pass
 
 
+class TestTensorsUtils:
+
+    dev_data_format_to_dtype = {
+        # forge.DataFormat.Bfp2: torch.float16,
+        forge.DataFormat.Bfp2: torch.bool,  # mapped
+        forge.DataFormat.Bfp2_b: torch.bfloat16,
+        forge.DataFormat.Bfp4: torch.float16,
+        forge.DataFormat.Bfp4_b: torch.bfloat16,
+        forge.DataFormat.Bfp8: torch.float16,
```

Comment:
I know that torch has 8bit FP as well: `torch.float8_e4m3fn`, however, seems like there is limited support for it. As that is the case, `torch.float16` is probably the smallest general dtype we can use (next to the bool and int variants). 

In sum, current implementation is valid.

Reply:
Torch data formats are now restricted to float32 and int32. It looks like they are working fine and others like bfloat16, float16, int16, uint8, int8 are causing many issues.